sqlglot/dialects/prql.py

```diff
@@ -81,6 +85,36 @@ def _parse_take(self, query: exp.Query) -> t.Optional[exp.Query]:
             num = self._parse_number()  # TODO: TAKE for ranges a..b
             return query.limit(num) if num else None
 
+        def _parse_filter(self, query: exp.Select) -> t.Optional[exp.Query]:
+            if self._match(TokenType.L_PAREN):  # TODO: FILTER ( expr1 || expr2 && ...)
+                filter = self._parse_conjunction()
+                if not self._match(TokenType.R_PAREN):
+                    self.raise_error("Expecting )")
+            else:
+                filter = self._parse_conjunction()
+            return query.where(filter) if filter else None
+
+        def _parse_set_op(
+            self,
+            query: exp.Query,
+            append: bool = False,
+            intersect: bool = False,
+            remove: bool = False,
+        ) -> t.Optional[exp.Query]:
+            from_ = self._parse_from(skip_from_token=True)
```

Comment:
Why do we `_parse_from` here? Don't we only need the table part (`_parse_table`)?